src/functions-and-arrays.js

```diff
@@ -1,19 +1,98 @@
+/* ------------------------------------------------------------------------------------------------------------------------------------------------------ */
+// ATTEMPTED 23.03.2021 //
+
 // Iteration #1: Find the maximum
 
+let num1;
+let num2;
+
+function maxOfTwoNumbers(num1, num2) {
```

Comment:
Good stuff. Perhaps Google "getting the max of two numbers in JavaScript" to see other (more minimal) solutions.